examples/field_examples/viking_graben_line12/fwi/README.md

```diff
@@ -0,0 +1,17 @@
+# Full Waveform Inversion
+
+FWI is done using JUDI.
+
+The idea is the following:
+
+1. prepare initial model
```

Comment:
Start with line with capital letter. 

For 2 and 3, in practice that would not scale as you would have a massive data copy so you would probably just do "on-the-fly" filtering of a given shot record after reading it. Would be done using `judiFilter` in JUDI. How much work would it be to do that instead?

I also see that you use a slightly different filter than the one I have setup but could still wrap it in a preconditonner fairly easily

Reply:
Didn't know that. If `judiFilter` allows to apply the same filter to the source signal and shot records then it should work. I will try that.
One note is when we are trying to find minimal possible frequency we must be confident that the data (reflections/turnings waves) are still present after filtration. If we set low pass filter at 3 Hz for example then we may not see useful waves but only noise instead. Probably this can be added to example as a comment.